packages/angular_devkit/build_angular/src/angular-cli-files/utilities/service-worker/index.ts

```diff
@@ -14,14 +14,20 @@ import * as fs from 'fs';
 import * as semver from 'semver';
 
 import { resolveProjectModule } from '../require-project-module';
-import { map, reduce, switchMap, concatMap, mergeMap, toArray, tap } from "rxjs/operators";
+import { map, reduce, switchMap, concatMap, mergeMap, toArray, tap, filter } from "rxjs/operators";
 import { Observable, merge, of, from } from "rxjs";
 
 
 export const NEW_SW_VERSION = '5.0.0-rc.0';
 
 
 class CliFilesystem implements Filesystem {
+  private _augmentedSwPaths = [
```

Comment:
Maybe extract this to a `const` instead of a private field, to further indicate these are hardcoded paths.